back/internal/dbg/dbg_test.go

```diff
@@ -0,0 +1,38 @@
+package dbg
+
+import (
+	"io/ioutil"
+	"testing"
+)
+
```

Comment:
Please add a surface test that ensures that the package functions Log and Logf do not print until SetOut is called with a valid io.Writer (consider using bytes.Buffer), and that Log/Logf can be shut off by calling SetOut with nil.